src/planner/join_order_enumerator.cpp

```diff
@@ -164,17 +164,20 @@ void JoinOrderEnumerator::planRelScan(uint32_t relPos) {
     newSubgraph.addQueryRel(relPos);
     auto predicates = getNewlyMatchedExpressions(
         context->getEmptySubqueryGraph(), newSubgraph, context->getWhereExpressions());
-    for (auto direction : REL_DIRECTIONS) {
+
+    std::vector<ExtendDirection> EXTEND_DIRECTIONS = {ExtendDirection::FWD, ExtendDirection::BWD};
```

Comment:
inline l168 and l169
`for (auto direction : {ExtendDirection::FWD, ExtendDirection::BWD})` 

And leave a comment saying something along the line of, `regardless whether rel is directed or not, we always enumerate two plans, one from src to dst, and the other from dst to src`